lib/kvdag/attrnode.rb

```diff
@@ -1,7 +1,19 @@
 class KVDAG
+
+  # Mixin with common methods for managing the +attrs+ of
+  # vertices and edges in a KVDAG.
+
   module AttributeNode
     attr_reader :attrs
 
+    # Returns the value for an +attr+. If the +attr+ can't be found,
+    # it will raise a KeyError exception.
+    #
+    # +options+:
+    #   shallow: If true, lookup is limited to attrs defined in
+    #            this attrnode. The default is lookup in the tree
+    #            returned by +to_hash_proxy+.
+
```

Comment:
This looks really weird, with the entire block being verbatim text. Consider instead the following:
```
+options+:
[+shallow+] Lorem ipsum
            dolor sit amet
```
which makes a definition list.